gettsim/benefits/alg2.py

```diff
@@ -221,50 +221,132 @@ def grossinc_alg2(household):
     )
 
 
-def e_anr_frei(household, params):
+def e_anr_frei_2005_01(household, params):
     """Calculate income not subject to transfer withdrawal for the household.
 
+    Legislation in force 2005-01-01 to 2005-09-30.
+
+    Determine the gross income that is not deducted. Withdrawal rates depend
+    on monthly earnings. § 30 SGB II."""
+
+    cols = ["m_wage", "ekanrefrei", "incometax", "soli", "svbeit"]
+    household.loc[:, cols] = household.groupby("pid")[cols].apply(
+        e_anr_frei_person_2005_01, params, params["a2eg3"]
+    )
+
+    return household
+
+
+def e_anr_frei_person_2005_01(person, params, a2eg3):
+    """Calculate income not subject to transfer withdrawal for each person.
+
+    Legislation in force 2005-01-01 to 2005-09-30."""
```

Comment:
```suggestion
    Legislation in force 2005-01-01 to 2005-09-30.
    
    """
```

Reply:
This is very nitpicky because there is not much to say to this excellent PR. Normally you would leave a blank line at the end of a longer docstring and directly follow-up with code on the next line after the closing ``"""``.